graphics.py

```diff
@@ -210,7 +211,34 @@ def qso_classes_graph(size, qso_classes):
         values.append(d[0])
     return make_pie(size, values, labels, "QSOs by Class")
 
+def qso_table(size, qsos):
+    """
+    create the a table of the qso log
+    """
+    if len(qsos) == 0:
+        return None, (0, 0)
 
+    count = 0
+    cells = [['Time', 'Call', 'Band', 'Mode', 'Operator', 'Section', 'Station']]
+    
+    for d in qsos:
```

Comment:
since the number of qsos is already limited by the fetch, does this need to also limit the size?  Another way to do this would be to slice the QSOs e.g. `for d in qsos[:10]` or something like that.  in all this looks good.  TY

Reply:
Good point. Since the fetch of the records is its own discrete function, this protects the table generation in case a result set of too large a size is returned. That way the generation protects itself without relying upon how big the supplied set is. I wasn't aware of the syntax for d in qsos[:10]. That is handy.